.github/workflows/release_candidate_checklist.yml

```diff
@@ -0,0 +1,44 @@
+name: Release conductor checklist
+
+on:
+  pull_request:
+
+jobs:
+  release-template:
+    # Change to changeset-release/main before PR is merged
+    if: ${{ github.head_ref == 'rezrah/release-checklist' }}
+
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
+
+      - name: Get or Create Comment
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const fs = require('fs')
+            const body = await fs.readFileSync('.github/release_template.md', 'utf8')
+            console.log('body', body)
```

Comment:
Is this intentionally left in, or just debugging?

Reply:
Yeah, this is just debugging and will be removed before merging 👍